_overviews/collections-2.13/trait-iterable.md

```diff
@@ -22,7 +22,7 @@ Collection classes that implement `Iterable` just need to define this method; al
 
 * **Addition**, `concat`, which appends two collections together, or appends all elements of an iterator to a collection.
 * **Map** operations `map`, `flatMap`, and `collect`, which produce a new collection by applying some function to collection elements.
-* **Conversions** `toArray`, `toList`, `toIterable`, `toSeq`, `toIndexedSeq`, `toStream`, `toSet`, `toMap`, which turn an `Iterable` collection into something more specific. All these conversions return their receiver argument unchanged if the run-time type of the collection already matches the demanded collection type. For instance, applying `toList` to a list will yield the list itself.
+* **Conversions** `to`, `toList`, `toVector`, `toMap`, `toSet`, `toSeq`, `toIndexedSeq`, `toBuffer`, `toArray` which turn an `Iterable` collection into something more specific. If the destination is a mutable collection(to(collection.mutable.X), to(collection.X), toArray, toBuffer), a new collection is created by copying the original elements. All these conversions return their receiver argument unchanged if the run-time type of the collection already matches the demanded collection type. For instance, applying `toList` to a list will yield the list itself. 
```

Comment:
looks like some missing backquotes here, to make `to(collection.mutable.X)` and so forth render in monospace font

Reply:
thanks!
fixed it.